apps/skilavottord/ws/src/app/modules/recyclingRequest/recyclingRequest.service.ts

```diff
@@ -110,6 +120,31 @@ export class RecyclingRequestService {
     return res
   }
 
+  async findRecyclingRequestWithPermno(
+    permno: string,
+    user: User,
+  ): Promise<RecyclingRequestModel[]> {
+    this.logger.info(
+      `---- Starting findRecyclingRequestWithPermno for ${permno} ----`,
+    )
+    const carList = await this.samgongustofaService.getVehicleInformation(
```

Comment:
`getVehicleInformation` implies that it needs permno to read it ... but this is actually `getUsersVehicles`